index.js

```diff
@@ -1,117 +1,142 @@
 const express = require("express");
+const morgan = require("morgan");
+const bodyParser = require("body-parser");
+const uuid = require("uuid");
+
 const app = express();
-const morgan = require("morgan")
-bodyParser = require('body-parser'),
-      uuid = require('uuid');
-
-    const app = express();
-    app.use(bodyParser.json());
-
-    let topMovies = [
-{
- id: 1,
- title: 'harry potter and the sorcerer\'s stone',
- author: 'J.K. Rowling',
- director: 'this person',
- genre: 'this genre'
-},
-{
- id: 2,
- title: 'Lord of the rings',
- author:'J.R.R. Tolkien',
- director: 'this person',
- genre: 'this genre'
-},
-{
- id: 3,
- title: 'Twilight',
- author: 'stephanie Meyer',
- director: 'this person',
- genre: 'this genre'
- }
-];
+app.use(morgan("common"));
```

Comment:
Move morgan all the way here so it captures any incoming requests including those related to static files